### ISSUE TITLE
Improvement in Home/release/Ocaml 4.12.0 page

### ISSUE THREAD

**Issue by shreyaswikriti** [ocaml/v2.ocaml.org#1460]
![image](https://user-images.githubusercontent.com/57635473/113767925-dd34f780-973c-11eb-9243-bff41ca57a82.png)

since the content is very large, we can provide users navigation-type Content. , So that users don't have to scroll down all the page to see the topics. They can easily navigate through the contents. For more reference see the below image. @patricoferris
![image](https://user-images.githubusercontent.com/57635473/113769284-70226180-973e-11eb-97de-eb7b0172fb3b.png)

**Comment by pitag-ha** [comment]
That's a great idea, thanks! :)

Just as a reference: there is another issue that already adds a ToC to another page: #1385.

@shreyaswikriti, Outreachy applicants can only work on 2, max 3 simple issues (see [here](https://github.com/ocaml/ocaml.org/issues/1245#issue-821019972)). From the top of my head I'd say you've already reached that. Is that right?

**Comment by shreyaswikriti** [comment]
No, I am working on only one issue right now.

**Comment by pitag-ha** [comment]
> No, I am working on only one issue right now.

I've already seen several great ideas of yours :) What about #1431 and #1454 for example? Are you not working on them? By the way, I'm not sure if #1431 classifies as simple or not (cc @patricoferris ).

**Comment by shreyaswikriti** [comment]
yes surely I am working on the #1454 issue but I was not sure if the #1431 issue will get approved by you. That's why I didn't count it. By the way @pitag-ha  I have come across many UX and Usability issues on the webpage. For this, I have made a doc where I have addressed issues and possible solutions. Kindly have a look.  Thanks.
https://docs.google.com/presentation/d/1cF_QoT3KwNF4lbnLhqU7xwNvlhO9QThl0fa0iRbGi8I/edit#slide=id.g7a202b835a_0_17

I have presented my ideas with diagrams etc.

**Comment by pitag-ha** [comment]
> For this, I have made a doc where I have addressed issues and possible solutions. Kindly have a look. Thanks.
https://docs.google.com/presentation/d/1cF_QoT3KwNF4lbnLhqU7xwNvlhO9QThl0fa0iRbGi8I/edit#slide=id.g7a202b835a_0_17

That's great, @shreyaswikriti, thanks! I'm not a mentor for the "Improve ocaml.org" project. Let me cc @patricoferris adn @gs0510 to see the doc (although this issue is actually not the right place for that!)

**Comment by shreyaswikriti** [comment]
Thanks, @pitag-ha, Sorry for mentioning this issue here. Should I open a particular issue for this? I am new to the open-source community, that's why don't know more about this. Looking forward to your guidance.

**Comment by patricoferris** [comment]
Hi @shreyaswikriti, 

Wow, that document is very impressive, thank you for making it! In general the best approach for tackling problems or making suggestions is to do it incrementally or in a "divide and conquer" fashion. This makes it much easier for us to give feedback on a specific, narrowly-scoped issue rather than a larger, more general set of problems. 

My advice would be to extract these smaller, narrowly-scoped issues from your document and create them here as a single issue (re-use the excellent diagrams!). Then we can provide feedback more easily and quickly. Thanks again for the contributions :))

**Comment by shreyaswikriti** [comment]
@patricoferris, thanks for your kind instructions. I'll open small issues with the help of this doc.

**Comment by shreyaswikriti** [comment]
@patricoferris, Kindly review the PR attached
